lib/globalize.js

```diff
@@ -7,5 +7,10 @@
 var path = require('path');
 var SG = require('strong-globalize');
 
+if (process.env.NODE_ENV === 'test') {
+  module.exports = SG({language: 'en'});
```

Comment:
To set the default language, you can use two environment variables:
STRONGLOOP_GLOBALIZE_APP_LANGUAGE, or the OS-level local environment variable, e.g. LANG.